File: JavaScript/1156.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split('\n');

let soma = 0;
let dividendo = 1, divisor = 1;
let constante = 2;


while(true){
    if(dividendo >= 39){
        break;
    }
    
    soma = soma + (dividendo / (divisor * 1.0));

    dividendo = dividendo + constante;
    divisor = divisor * constante;
}

console.log(soma.toFixed(2))

File: C++/1156.cpp
#include<iostream>
#include<iomanip>

using namespace std;

int main(){
    int constante = 2;
    double soma = 0;
    double dividendo = 1, divisor = 1;

    while(true){
        if(dividendo >= 39){
            break;
        }

        soma = soma + (dividendo / (1.0 * divisor));

        dividendo += constante;
        divisor *= constante;
    }

    cout << fixed << setprecision(2);
    cout << soma << endl;

    return 0;
}